package.json

```diff
@@ -17,7 +17,8 @@
     "site": "rimraf tmp/site && webpack --env.site --env.element=blaze-elements",
     "commit": "git-cz",
     "commitmsg": "validate-commit-msg",
-    "precommit": "yarn ts:lint && yarn test",
+    "precommit": "npm run ts:lint",
```

Comment:
Haven't you forgot to delete it?

Title says move, but tslit is still there :-)